doc/developer/setup/index.md

```diff
@@ -172,6 +172,26 @@ See [AWS S3 Bucket Storage Setup](https://docs.aws.amazon.com/AmazonS3/latest/us
 
 There is no other configuration necessary in IRIDA to use cloud based storage. After adding these values to the configuration file you should be able to start up IRIDA, and it will use the cloud based storage that is configured.
 
+There isn't a local storage emulator available from Amazon AWS but we can make use of localstack to develop locally. The best way to make use of localstack is to use it with a docker container.
+
+1) Install `docker` and the `awslocal-cli`.
```

Comment:
Isn't it awscli or awscli-local?

Reply:
You are correct! Got it mixed up with the commandline command awslocal. Thanks. Updated in dae6691d97fc7d47f685d824cb937955786cb866